Directory.Build.props

```diff
@@ -1,18 +1,33 @@
 <Project>
+    <!-- Make F# support Central Package Management -->
     <PropertyGroup>
-        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <DisableImplicitSystemValueTupleReference>true</DisableImplicitSystemValueTupleReference>
+        <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    </PropertyGroup>
+    <!-- Use lock files -->
+    <PropertyGroup Condition="'$(GITHUB_ACTIONS)' == 'true'">
+        <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
```

Comment:
If you add the `Dotnet.Reproduciblebuilds` package this setting of `ContinuousIntegrationBuild` get done for you automatically - I co-maintain that package and would recommend that.